### ISSUE TITLE
Could RT be added to Half-Life mods like Team Fortress Classic?

### ISSUE THREAD

**Issue by Nomad7** [sultim-t/xash-rt#38]
My video card doesn't support ray tracing, but I would have to upgrade if I could play TFC with ray tracing :D

I'm kind of dumb about the mechanics involved here - this doesn't work for HL mods by default, right? It has to be specifically added?

**Comment by sultim-t** [comment]
Mods need to be recompiled to x64 to load them. But even after that, their maps and textures need to be readjusted.
For now only the original Half-Life 1 campaign is available

**Comment by crsleeth** [comment]
@sultim-t Would that include things like Sven co-op?

**Comment by nekonomicon** [comment]
@crsleeth Nope. Sven-coop is closedsource and the latest steam version uses custom goldsource engine with incompatible interfaces.
Only mods from following lists will be possible to run on alternative architectures:
https://github.com/FWGS/xash3d-fwgs/blob/master/Documentation/supported-mod-list.md
https://github.com/FWGS/xash3d-fwgs/blob/master/Documentation/opensource-mods.md
https://github.com/FWGS/hlsdk-portable/wiki/Mods

**Comment by palaashatri** [comment]
I'm wondering if Half Life expansions Blue Shift and Opposing Force would run with this? I don't think xash3d team is focusing on them, right?

Just throwing it out there, if someone wants to try, y'know?

**Comment by jurassicjordan** [comment]
> I'm wondering if Half Life expansions Blue Shift and Opposing Force would run with this? I don't think xash3d team is focusing on them, right?
> 
> Just throwing it out there, if someone wants to try, y'know?

I tried running bshift and it errored, saying "cant initalise client_amd64.dll"  so probably not. however I'm running it on linux thru proton so it could be working on windows but doesnt seem as such

**Comment by thedirptastic** [comment]
> > I'm wondering if Half Life expansions Blue Shift and Opposing Force would run with this? I don't think xash3d team is focusing on them, right?
> > Just throwing it out there, if someone wants to try, y'know?
> 
> I tried running bshift and it errored, saying "cant initalise client_amd64.dll" so probably not. however I'm running it on linux thru proton so it could be working on windows but doesnt seem as such

I just copied it from a different folder and the game booted up, but the campaign button was greyed out.
